### DISCUSSION TITLE
新版网易云桌面版3.0跟你重构的界面样啊真的6

### DISCUSSION THREAD

**Discussion by arsnyeon** [SPlayer-Dev/SPlayer#86] [讨论 General]
![QQ截图20231115182904](https://github.com/imsyy/SPlayer/assets/41101004/96c9438b-3ec2-43d4-a6e8-3a03857de19f)

**Comment by imsyy** [comment]
🤣 我这差远了

**Reply by Takenforgranted** [reply]
天下文章一大抄，人家开发有时候犯懒直接用你的了也正常。
